packages/compass/src/main/telemetry.ts

```diff
@@ -121,7 +121,7 @@ class CompassTelemetry {
   }
 
   private static _flushTelemetryAndIgnoreFailure() {
-    return this.analytics?.flush().catch(() => Promise.resolve());
+    return this.analytics?.closeAndFlush().catch(() => Promise.resolve());
```

Comment:
Out of curiosity - it won't be any different from just having this -
```
return this.analytics?.closeAndFlush().catch(() => {});
```
right?

Reply:
none that i can tell yeah